agreements/activity-types.md

```diff
@@ -0,0 +1,19 @@
+# Activity Types Agreement
+
+We have assessed that there are four types of activities that members carry out within Root Systems:
+
+## External
+
+This is contracting work with the outside world - usually software development, but could include any other types of consulting work.
+
+## Internal Investment
+
+This is either internal work towards development of a Root Systems product, or personal development / upskilling of a member (i.e. conferences, training).
+
+## Internal Administration
+
+This is time spent towards the administration of the business - typically work done as part of either a [coordinator](../roles/coordinator) or [director](../roles/director) role.
```

Comment:
@ahdinosaur basically coordination and directors are internal admin types. The main point of this is that I realised that there is a difference between investment work and administrative work. one is a nice to have and the other just needs to happen. I haven't thought of any rules around this but rather wanted to differentiate between the two. eg at buffer level 0 we have to have admin but don't have to have dogstack/cobuy/freehold work. this is an oversimplification. but the idea was to be able to differenciate the two. We could also then look to put limits on how much internal administration we want. ie i think we are blowing the budget at the mo. 

The other thing is to be able to look at how we value (financially) internal administration work. This is a conversation I would like to have at some point. So splitting it out made sense for that. sorry it came out of nowhere.